utils/formats/table.go

```diff
@@ -4,6 +4,17 @@ package formats
 // Annotations are as described in the tableutils.PrintTable description.
 // Use the conversion methods in this package to convert from the API structs to the table structs.
 
```

Comment:
Lets consider fixing some fields names in this file. We have 2 kinds of Table Rows we have in structs here- directDependenciesTableRow and directPackagesTableRow.
In licenseScanTableRow, licenseViolationScanTableRow, operationalRiskViolationScanTableRow we have field named directDependencies although its of kind directPackagesTableRow
(In vulnerabilityScanTableRow for example a similar field is called directPackages)